integration_tests/src/main/python/parquet_test.py

```diff
@@ -1145,3 +1145,23 @@ def test_parquet_read_encryption(spark_tmp_path, reader_confs, v1_enabled_list):
         lambda: with_gpu_session(
             lambda spark: spark.read.parquet(data_path).collect(), conf=conf),
         error_message='The GPU does not support reading encrypted Parquet files')
+
+def test_parquet_read_count(spark_tmp_path):
+    parquet_gens = [byte_gen, short_gen, int_gen, long_gen, float_gen, double_gen,
```

Comment:
Nit: Can simply to [int_gen, string_gen, double_gen], because will not touch any column data.